src/ol/source/wms.jsdoc

```diff
@@ -1,3 +0,0 @@
-/**
- * @namespace ol.source.wms
```

Comment:
You'll have to rebase this pull request anyway before merging. Maybe you get a chance to verify that then.